opal/models.py

```diff
@@ -54,6 +58,38 @@ def deserialize_date(value):
     return dt.date()
 
 
+def generate_prefixes_category_or_list(
+    method_name, patient_list=None, episode_category=None
+):
+    """ this method is used to translate the deprecated method
+        of passing in patient list or category to
+        Subrecord.get_display_template
+        Subrecord.get_detail_template
+        Subrecord.get_form_template
+
+        into the new method of passing in a prefix.
+        It also warns the user about the deprecation
+    """
+
+    warnthem = """
+    opal.models.Subrecord.{} no longer takes
+    patient_list or episode_type.
```

Comment:
Well, it does right... It _won't_ take them :)

I think all these messages should have a target version at which they will be removed?